packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.Props.ts

```diff
@@ -88,6 +88,20 @@ export interface IComboBoxProps extends ISelectableDroppableTextProps<IComboBox>
    * in case an option does not come with user-defined custom styles
    */
   comboBoxOptionStyles?: Partial<IComboBoxOptionStyles>;
+
+  /**
+   * Callout width is based on parent width
+   * When set to false the callout will be sized based on the width of the callout's content
+   * @default true;
+   */
+  calloutFixedWidth?: boolean;
+
+  /**
+   * Set max height of callout
+   * When not set the callout will expand with contents up to the bottom of the screen
+   */
+  calloutMaxHeight?: number;

```

Comment:
You don't need this anymore, right? You're just passing it through calloutProps.

Reply:
Hi! It looks like with my new update with the new prop (calloutMaxHeight) is no longer getting passed through from the comboBox to the callout.
I think I need to add it back the way it was?
![untitled](https://user-images.githubusercontent.com/25463697/30758845-d9548392-9f88-11e7-85d2-98e48d341e45.jpeg)

If I remove the prop from ComboBox.Props, I don't see how I would be able to pass the height from the ComboBox component to the Callout component.

Reply:
the user would do <ComboBox calloutProps={{calloutMaxHeight: '400px'}} /> 

and then that would be passed through via { ...calloutProps } in the callout.

Reply:
This is really great!  I will update, thanks!